euphoria-operator-testkit/src/main/java/cz/seznam/euphoria/operator/test/accumulators/GetSnapshot.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2016-2017 Seznam.cz, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.seznam.euphoria.operator.test.accumulators;
+
+interface GetSnapshot<V> {
```

Comment:
I'd suggest naming the method `apply` - `GetSnapshot#apply` reads better than `GetSnapshot#getSnapshot`.